ash/src/extensions/ext/full_screen_exclusive.rs

```diff
@@ -77,8 +70,12 @@ impl FullScreenExclusive {
             .result_with_success(present_modes)
     }
 
+    pub fn name() -> &'static CStr {
+        vk::ExtFullScreenExclusiveFn::name()
+    }
+
     pub fn fp(&self) -> &vk::ExtFullScreenExclusiveFn {
```

Comment:
Perhaps you should unify the method name and the field name if you're renaming it anyway.

Reply:
Should I rename the function to `fns()` or the field name to `fp` ?

Reply:
The field name is purely internal, so I don't think it matters much what it is. The functions should not be renamed as that would be gratuitous breakage.

Reply:
@Ralith I think `fp` works then, time to sign-off and merge this PR?

Reply:
That particular change seems fine to me, yeah. I don't know if I'll have the energy to intensively review this whole PR in the short term, but considering it's straightforward refactoring I see no reason for you to wait on me if you've already done so.

Reply:
I have previously gone through the entire PR and checked that there are no unintended changes besides what we discussed and what's listed in the changelog. Will do so again with the `fp` changes and new changelog, validate how this all works against one of our Vulkan applications that heavily uses a bunch of extensions, and perhaps just merge and perform a release after that.